shard.lua

```diff
@@ -810,41 +831,31 @@ local function get_server_list()
     return configuration.servers
 end
 
-local function is_shard_connected(uri, conn)
-    local try = 1
-    while try < 20 do
-        if conn:ping() then
-            return true
-        end
-        try = try + 1
-        fiber.sleep(0.01)
-    end
-    return false
-end
-
 local function shard_connect(s_obj)
-    if s_obj.conn == nil then
-        return false
-    end
-    if s_obj.conn:is_connected() then
-        maintenance[s_obj.id] = nil
+    -- check if connection exists
+    if s_obj.conn ~= nil and s_obj.conn:is_connected() then
+        if maintenance[s_obj.id] ~= nil then
+            maintenance[s_obj.id] = nil
+            log.info("Return '%s' from maintenance.", s_obj.uri)
```

Comment:
Get rid of dots in the end of the string. They don't have any meaning in the log file

Reply:
In all other places - too